test-cases/nemesis/template-longevity-10gb-3h.yaml.j2

```diff
@@ -0,0 +1,24 @@
+test_duration: 255
```

Comment:
I guess all my comments on the yaml should have been here...
We need to change also the test_duration.